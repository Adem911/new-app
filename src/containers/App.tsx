import React from 'react';
import { BrowserRouter, Route } from 'react-router-dom';
import './App.css';
import NavigationBar from '../components/navigation-bar/NavigationBar';
import HomePage from './home-page/HomePage';
import PageOne from './page-1/PageOne';
import PageTwo from './page-2/PageTwo';
import PageThree from './page-3/PageThree';


const App = () => {
  return (
    <div className="App">
      <BrowserRouter>
        <NavigationBar />
        <div>
        <Route path='/' exact component={HomePage} />
        <Route path='/page1' exact component={PageOne} />
        <Route path='/page2' exact component={PageTwo} />
        <Route path='/page3' exact component={PageThree} />

        </div>
      </BrowserRouter>
    </div>
  );
}

export default App;
