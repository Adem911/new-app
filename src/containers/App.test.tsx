import React from 'react';
import App from './App';
import { shallow } from 'enzyme';


describe('<App />', () => {
  it('renders one h1 element', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  })
})
