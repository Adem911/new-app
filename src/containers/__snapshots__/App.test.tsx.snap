// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders one h1 element 1`] = `
<div
  className="App"
>
  <BrowserRouter>
    <NavigationBar />
    <div>
      <Route
        component={[Function]}
        exact={true}
        path="/"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/page1"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/page2"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/page3"
      />
    </div>
  </BrowserRouter>
</div>
`;
